packages/types/src/api/ontime-controller/BackendResponse.type.ts

```diff
@@ -21,6 +21,11 @@ export type ProjectFile = {
 
 export type ProjectFileList = Array<ProjectFile>;
 
+export type ProjectFileListResponse = {
+  files: ProjectFileList;
+  lastLoadedProject: string | null;
```

Comment:
There always has to be an open project. in which cases there is not?
How do we handle not having a project? do we create one ourselves? what is it called?

Reply:
That's a very good point, for now when we never import a project we just use the regular `db.json`, and `lastLoadedProject` is `null`.

But maybe the best solution would be to create a `default.json` (maybe thats a decent name?) instead

Reply:
yes, I dont see an issue with it. The user would be able to rename it if they want.
Could we run into trouble with potential duplicate names?

Reply:
perhaps, maybe we could have a warning if the user tries to upload a file with a name that already exists on the uploads folder?

if we want to avoid duplicates on the default one, we could have a timestamp on the default filename maybe?

Reply:
I dont have too many opinions here, I am sure it is ok.
For the backend upload we could just add in increment like your OS does `default (1).json`

Reply:
will do that, I've been trying for the past hour or so but have ran into some issues, I'll try again tomorrow with a clear mind

Reply:
sure, I would suggest that you keep in simple for now, we can refine it later
it is ok with me if you just thrown an error if the file exists, we can deal with it later

Reply:
made it so it throws an error for now. I had logic working for it but it had a bug that it would also modify the previous file

commit for reference is https://github.com/cpvalente/ontime/pull/679/commits/b896a9b5f543b9e3bc49a9e79decc5ca43c2c780

we can revisist this later